cluster/charts/rook-ceph/templates/prometheus-ceph-v14-rules.yaml

```diff
@@ -0,0 +1,257 @@
+{{- if .Values.monitoring.enable }}
+apiVersion: monitoring.coreos.com/v1
```

Comment:
How can we avoid duplication with what is in the exampes/ceph/monitoring folder? For example, can we use a `.Files.Glob` to point to those files? If needed, a github symlink could be added from this folder to the original folder.

Reply:
About using `Files.Glob` it can only be used from the charts dir.

About creating GitHub symlink from chart dir to the outside, I don't think this will help non-helm users because tools not using helm will not work due to the template metadata the files have. Non-helm users need plain yaml files so that kubectl or other tools can parse them.

To reduce duplication and eventual inconsistencies there could be a `make` target which generates the files from helm chart.

Reply:
> To reduce duplication and eventual inconsistencies there could be a make target which generates the files from helm chart.

:+1: